test/integration/targets/yum/tasks/yum.yml

```diff
@@ -92,6 +92,20 @@
         - "yum_result is success"
         - "not yum_result is changed"
 
+- name: install sos again with missing repo enablerepo
+  yum: name=sos state=present enablerepo="thisrepodoesnotexist"
+  ignore_errors: True
+  register: yum_result
+- name: verify no change on fourth install with missing repo enablerepo (yum)
```

Comment:
@mattclay @abadger I'm curious on your opinions here, the YUM4(dnf) and dnf cli both handle this gracefully (Fedora and RHEL8+) but the YUM3(real yum) cli fails in this scenario so I've modeled each backend module (`yum` and `dnf` respectively) to mirror their cli behavior but that makes this test case feel odd. Thoughts?

Reply:
@maxamillion I must be missing something here. I thought the original bug report was that the yum command on the cli doesn't fail, but the Ansible yum module does. That's different than what you describe.